NEMO/static/nemo.js

```diff
@@ -96,6 +96,78 @@ function collapse_all_categories()
 	save_sidebar_state();
 }
 
+function get_qualified_tools_for_user(user_id) {
+	ajax_get("/api/users/"+user_id+"/", {}, (data) => {
+		localStorage.setItem("qualifiedTools", data.qualifications)
+	})
+}
+
+function toggle_qualified_tools() {
+	// Toggle local storage data value
+	if (localStorage.getItem("showQualifiedTools") === "true") {
+		localStorage.setItem("showQualifiedTools", "false");
+	} else {  // Item showQualifiedTools is 'false' or not set.
+		localStorage.setItem("showQualifiedTools", "true");
+	}
+
+	set_qualified_tools_button_status(
+		localStorage.getItem("showQualifiedTools") === "true"
+	)
+
+	update_tool_list_display("toggle");
+	if (localStorage.getItem("showQualifiedTools") === "true") {
+		hide_empty_tool_categories();
+	} else {
+		show_all_tool_categories();
+	}
+}
+
+function update_tool_list_display(item_function) {
+	// Retrieve the list of tools the user is qualified for.
+	let availableToolList = localStorage.getItem("qualifiedTools").split(",");
```

Comment:
I am getting an error here, if I clear the localStorage `localStorage.clear()` then go to either tool control or calendar page.
error is `Cannot read properties of null (reading 'split')`

Reply:
Not storing the list of qualified tools in localStorage anymore